segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/StreamSegmentContainerTests.java

```diff
@@ -1532,6 +1532,7 @@ public void testForcedMetadataCleanup() throws Exception {
                 .builder()
                 .with(ContainerConfig.SEGMENT_METADATA_EXPIRATION_SECONDS, (int) DEFAULT_CONFIG.getSegmentMetadataExpiration().getSeconds())
                 .with(ContainerConfig.MAX_ACTIVE_SEGMENT_COUNT, maxSegmentCount + EXPECTED_PINNED_SEGMENT_COUNT)
+                .with(ContainerConfig.STORAGE_SNAPSHOT_TIMEOUT_SECONDS, (int) DEFAULT_CONFIG.getSegmentMetadataExpiration().getSeconds())
```

Comment:
Just checking if this is intended - did you mean `.getSnapshotTimeout()`?

Reply:
Fixed.